packages/gdl-frontend/locale/en/messages.json

```diff
@@ -12,26 +12,66 @@
     "translation": "<0>Selected: </0>{0}",
     "origin": [
```

Comment:
This file aren't supposed to be committed, or?

Reply:
For now it should, will remove it!